schema/cassandra/cadence/versioned/v0.12/events_v2.cql

```diff
@@ -25,4 +25,17 @@ CREATE TABLE history_tree (
   PRIMARY KEY ((tree_id), row_type, branch_id )
 ) WITH COMPACTION = {
 'class': 'org.apache.cassandra.db.compaction.LeveledCompactionStrategy'
-};
\ No newline at end of file
+};
+
+CREATE TYPE history_branch_info (
+  branch_token                     blob,
+  next_event_id                    bigint,
+  last_first_event_id              bigint,
+  history_size                     bigint,
```

Comment:
This is for history size for single branch?  Or it also accumulates all ancestor branches?  Please also include a comment.

Reply:
OK sure